developerportal/apps/search/templates/site-search.html

```diff
@@ -0,0 +1,37 @@
+{% extends "base.html" %}
+{% load wagtailcore_tags static %}
+
+{% wagtail_site as current_site %}
+
+{% block title %}Search - {{current_site.site_name}} {% endblock %}
+
+{% block body_class %}site-search{% endblock %}
+
+{% block content %}
+  {% static "img/icons/search-white.svg" as page_icon_asset_url %}
+  {% include "molecules/header-strip.html" with content="Search" element="h1" page_icon_asset_url=page_icon_asset_url %}
+
+  <div
+    class="
+      mzp-l-content
+      mzp-has-sidebar
+      mzp-l-sidebar-right
+      {% if not search_results %}no-results{% endif %}
+    ">
+
+    <main role="main" class="mzp-l-main custom-width">
+      {% include "organisms/search-results.html" with total_results=total_results search_query=search_query search_results=search_results %}
+    </main>
+
+    <aside class="mzp-l-sidebar custom-width">
+      <form class="site-search-form" action="{% url 'search.site_search' %}" method="get">
```

Comment:
I did this with no CSRF token here because it's a GET with no side effects – _however_ as I write this comment I'm remembering we to note search terms as "hits" for the search backend, so someone flooding us with bad search terms might distort things. However, we'd then end up with the CSRF token being passed as a GET querystring param, which is a bit mucky. Any opinions on this @Gregoor ? My preference right now is to go with tightness over prettiness

Reply:
Also, this is relevant: https://infosec.mozilla.org/guidelines/web_security#csrf-prevention

Reply:
I think for cross-site worries CSRF is more of a fallback at this point anyway, given that SameSite cookie support is at [~90%](https://caniuse.com/#feat=same-site-cookie-attribute) (and probably at 99.99% for developers visiting the site). So imo GET without CSRF is fine.

Then again, I don't know much about how that hit counting is working. Can you link me to some code I could use to read-up on it?

Reply:
https://github.com/wagtail/wagtail/blob/bb2e460c0b92fba802fc3f369730004d6b648e64/wagtail/search/models.py#L20 is where Query.add_hit() is defined. Looks like it "just" logs to a table/model, `QueryDailyHits`, but it doesn't look like much is done with that by default... https://github.com/wagtail/wagtail/search?q=QueryDailyHits&unscoped_q=QueryDailyHits

I'll check in with the Wagtail team and see what they say about this

Reply:
So it looks like it populates data for Wagtail's promoted search behaviour,  which we're not using in devportal. BUT it also tunes the results of some of the searches in the CMS, so it might be that an Editor could be shown less relevant results if there's a dedicated poisoning campaign, but I'm not sure how much of a real pain that'd be...

Reply:
I think, given we're not using promoted search (and have no imminent need to), there's no need to log search hits, so I'll just drop that line of code

- [ ] @stevejalim to drop hit counting